docs/output.md

```diff
@@ -29,6 +30,18 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 
 If `--save_intermediates` is specified, intermediate files generated by each process will be saved in the output directory.
 
+## Preprocessing
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bowtie_index/genome`: Cleaned genome.fa fasta.
+- `untar/bowtie_index`: Uncompressed bowtie index file.
+
+</details>
+
+Preprocessing is done to format reference files before using them in the workflow, it includes [untar](https://www.gnu.org/software/tar/manual/) and [bioawk](https://github.com/lh3/bioawk). If the bowtie_index file provided is in gzip format it will be processed by untar. The fasta file provided will be cleaned using bioawk.
```

Comment:
```suggestion
Preprocessing is done to format reference files before using them in the workflow, it includes [`untar`](https://www.gnu.org/software/tar/manual/) and [`bioawk`](https://github.com/lh3/bioawk). If the `bowtie_index` file provided is in gzip format it will be processed by `untar`. The fasta file provided will be cleaned using `bioawk`.
```